test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -622,3 +623,63 @@ test('wrong type of the blur event (GH-947)', function () {
     eventSimulator.blur(domElement);
     ok(raised);
 });
+
+module('Drag and Drop');
```

Comment:
Why you don't do?) You like a long list on test name more than a hierarchical structure of tests?